### PR TITLE
Catch correct parameter exception

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**

### CODE REVIEW

File: SumoForkClass.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ public function initErrbit()
                     $this->debug = $this->getContainer()->hasParameter('fork.debug');
                 }
                 $this->errbitApiKey = $this->getContainer()->getParameter('sumo.errbit_api_key');
-            } catch (\Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException $e) {
```
Comment (carakas) [inline SumoForkClass.php:120]: does ParameterNotFoundException extend from InvalidArgumentException?

Comment (jonasdekeukelaere) [inline SumoForkClass.php:120]: Yes, but the `getParameter()` function throws a `InvalidArgumentException` https://github.com/symfony/dependency-injection/blob/master/ContainerInterface.php#L78

Comment (carakas) [inline SumoForkClass.php:120]: Ok, just asked to make sure that we wouldn't get other exceptions by changing it

**Comment by WouterSioen** [comment]
looks good to me :shipit:

**Pull request merged**